mission.sqm

```diff
@@ -8,18 +8,18 @@ class EditorData
 	toggles=513;
 	class ItemIDProvider
 	{
-		nextID=1356;
+		nextID=1357;
```

Comment:
Please exclude `mission.sqm` from your commits unless you're making a deliberate change to it.  It causes unnecessary merge conflicts which need to be resolved and harms the readability of the `mission.sqm` change history.